tutor/src/screens/teacher-dashboard/plan-details.jsx

```diff
@@ -56,12 +57,17 @@ class CoursePlanDetails extends React.Component {
     plan: PropTypes.instanceOf(TeacherTaskPlan).isRequired,
   }
 
-  UNSAFE_componentWillMount() {
+  async UNSAFE_componentWillMount() {
     this.props.tourContext.otherModal = this;
+    // get scores to check if there are any students left to grade
+    await this.props.plan.scores.fetch();
+    await this.props.plan.scores.taskPlan.fetch();
+    await this.props.plan.scores.taskPlan.analytics.fetch();
+    await this.props.plan.scores.ensureExercisesLoaded();
   }
```

Comment:
@nathanstitt @jivey what you guys think about this? fetching the plan scores since it is not available yet when the instructor opens the plan-details. I am like...🤨

Reply:
If it turns out that we do need them, I'd suggest we change most to use `ensure Loaded` here and then on the `assignment-review/ux` so that they both won't fetch them twice